app/services/DateService.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import java.time.LocalDate
+import javax.inject.{Inject, Singleton}
+
+@Singleton
+class DateService @Inject()(appConfig: config.AppConfig) {
```

Comment:
Would you mind changing this to an `Object` instead, and the `now` function taking in app conf as an implicit parameter?

I suggest this because I'm currently adding the function to the `FinancialTransactions` model and it would be easier for it to simply call this functionality from a static object rather than having to DI the class in.